dal/src/main/java/com/laiteam/echowall/dal/entity/EmailInfo.java

```diff
@@ -0,0 +1,30 @@
+package com.laiteam.echowall.dal.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Component
+public class EmailInfo {
+    private String fromAddress;
+    private String toAddress;
+    private String subject;
+    private String body;
+
+    public static class EmailTemplate{
```

Comment:
why don't we 
1. Rename to EmailTemplate? 
2.add a static function = createVerficationTemplate?